### PR TITLE
aligned search button with search input field

### PR DESCRIPTION
**Pull request by Sarvesh-Ghildiyal**
Targets #{ISSUE_ID}
2282

### Description
Aligned search button with search input field

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: resources/views/knowledgecafe/library/books/index.blade.php [hunk 1]
```diff
@@ -22,7 +22,7 @@
             <input type="text" data-value="{{ request()->input('search') }}" 
             class="form-control" id="search_input" placeholder="search all books"
             v-model="searchKey">
-            <button class="btn btn-info ml-2" @click="searchBooks()">Search</button>
+            <button class="btn btn-info ml-2 col-md-py-1.5" @click="searchBooks()">Search</button>
```
Comment (Ayush8923) [inline resources/views/knowledgecafe/library/books/index.blade.php:25]: Why did you add the `col-py-md-1.5` class? And as far as I know, there is no such class for padding.

Reference link:- https://getbootstrap.com/docs/4.3/utilities/spacing/

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comment.

File: resources/views/knowledgecafe/library/books/index.blade.php [hunk 2]
```diff
@@ -22,7 +22,7 @@
             <input type="text" data-value="{{ request()->input('search') }}" 
             class="form-control" id="search_input" placeholder="search all books"
             v-model="searchKey">
-            <button class="btn btn-info ml-2" @click="searchBooks()">Search</button>
+            <button class="btn btn-info ml-2 col-py-md-1.5" @click="searchBooks()">Search</button>
```
Comment (Ayush8923) [inline resources/views/knowledgecafe/library/books/index.blade.php:25]: Why did you add the `col-py-md-1.5` class? And as far as I know, there is no such class for padding.

Reference link:- https://getbootstrap.com/docs/4.3/utilities/spacing/

File: Modules/Invoice/Resources/views/index.blade.php [hunk 3]
```diff
@@ -24,7 +24,7 @@
             <h4 class="mb-1 pb-1 fz-28">Invoices</h4>
             <span>
                 <a href="{{ route('invoice.create-custom-invoice') }}" class="btn btn-info text-white">Custom Invoice</a>
-                <a href="{{ route('invoice.create') }}" class="btn btn-success ml-2 text-white"><i class="fa fa-plus mr-1"></i>Add Invoice</a>
+                <a href="{{ route('invoice.create') }}" class="btn btn-info text-white">Add Invoice</a>
```
Comment (Ayush8923) [inline Modules/Invoice/Resources/views/index.blade.php:27]: Why did you make changes to this file? While there was no need to change this file.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comment.

File: Modules/Invoice/Resources/views/index.blade.php (hunk 3 above)
Comment (Sarvesh-Ghildiyal) [reply Modules/Invoice/Resources/views/index.blade.php]: i used _git diff command_ and found these changes, which i didnt make, so i changed the code as it was in master branch

File: resources/views/knowledgecafe/library/books/index.blade.php [hunk 4]
```diff
@@ -22,7 +22,7 @@
             <input type="text" data-value="{{ request()->input('search') }}" 
             class="form-control" id="search_input" placeholder="search all books"
             v-model="searchKey">
-            <button class="btn btn-info ml-2" @click="searchBooks()">Search</button>
+            <button class="btn btn-info ml-2 col-py-1.5" @click="searchBooks()">Search</button>
```
Comment (Ayush8923) [inline resources/views/knowledgecafe/library/books/index.blade.php:25]: Why did you add the `col-py-1.5` class? And as far as I know, there is no such class for padding.

Reference link:- https://getbootstrap.com/docs/4.3/utilities/spacing/

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comment.

**Review by Ayush8923 — APPROVED** [review]
Looks good.
